### PR TITLE
Adds and updates fields in LoRaWAN schema

### PR DESCRIPTION
**Pull request by laurensslats**
Fields added in LoRaWAN schema:
- manufacturerSuggestedRetailPrice
- firmwareConfigurability

Updated fields:
- keyProvisioning
- sensors

@johanstokking to review

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -706,6 +706,8 @@
     "endDevice": {
       "title": "End Device",
       "description": "LoRaWAN end device",
+      "manufacturerSuggestedRetailPriceEuro": "99",
+      "manufacturerSuggestedRetailPriceDollar": "120",
```
Comment (johanstokking) [inline schema.json]: This file is a schema with definitions of fields. This file shouldn't contain values, except for examples.

Comment (johanstokking) [reply schema.json]: Added MSRP via https://github.com/lorawan-schema/draft-devices/commit/80dc7204c371e1fa3345e00ef814856b2af7cf3e

**Review by johanstokking — COMMENTED** [review]
I made some changes to apply what I think you wanted to achieve: indications for how keys can be programmed, what features the firmware support and how firmware can be updated.

Please let me know if this is fine this way.

**Comment by laurensslats** [comment]
@johanstokking perfect.

**Pull request merged**